src/styles/quarks/_quarks.mixins.scss

```diff
@@ -245,6 +227,17 @@
 }
 
 
+/**
+ * FOCUS
+ * -------------------------------------------------------------------
+ */
+
+@mixin focus {
+  outline: 5px auto Highlight;
```

Comment:
wat is "Highlight"?

Reply:
Dit is de systeem highlight kleur (afhankelijk dus van je OS)